projects/netgrif-components-core/src/lib/view/tree-case-view/tree-component/case-tree.service.ts

```diff
@@ -360,6 +371,7 @@ export class CaseTreeService implements OnDestroy {
         const done = new ReplaySubject<void>(1);
 
         let params: HttpParams = new HttpParams();
+        params = params.set('size', this._treeConfiguration.pageSize + '');
```

Comment:
For better readability use string interpolation as on line below

Reply:
done